apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -117,7 +117,7 @@ var NetSimPacketEditor = module.exports = function (initialConfig) {
    * @type {Number}
    * @private
    */
-  this.maxPacketSize_ = initialConfig.maxPacketSize || Infinity;
+  this.maxPacketSize_ = initialConfig.maxPacketSize || 8192;
```

Comment:
Do we want 8192 vs. 1024?  If possible, I think we should go with the most conservative possible setting, make sure everyone is having a great experience, and then bump up from there.